transferui/src/test/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModelTest.java

```diff
@@ -271,6 +286,7 @@ public void testCreateQuoteTransfer_isSuccessful() {
         viewModel.setTransferNotes("Create quote test notes");
         viewModel.setTransferAmount("123.23");
 
+        viewModel.init();
```

Comment:
don't you want to verify which load is being called?

Reply:
FYI: Discussed. Implemented in some other method.